app/services/forest_liana/resources_getter.rb

```diff
@@ -16,7 +16,7 @@ def records
     end
 
     def count
-      search_query.count
+      @records.count
```

Comment:
Yes it is. We need to have the last updated @records (not just the search_query).